_pages/features.md

```diff
@@ -8,20 +8,29 @@ redirect_from: '/about.html'
 
 ## Key features
 
-**Data Catalogue** - The data catalogue in FAIRDOM-SEEK includes raw Datasets, [Standard Operating Procedures(SOPs)](https://fairdomhub.org/sops), [Models](https://fairdomhub.org/models), [Publications](https://fairdomhub.org/publications) and [Presentations](https://fairdomhub.org/presentations). All data are grouped by projects, and associated with the researchers who produced them. In order to encourage sharing of data we allow researchers flexibility in the formats they upload and share their data in. This means data formats in the FAIRDOM-SEEK catalogue can vary. We do offer a set of “best practice” guidelines for researchers who want to make their data available and usable to the widest possible audience.
+**Data Catalogue** – The data catalogue in FAIRDOM-SEEK includes raw Datasets, [Standard Operating Procedures(SOPs)](https://fairdomhub.org/sops), [Models](https://fairdomhub.org/models), [Publications](https://fairdomhub.org/publications) and [Presentations](https://fairdomhub.org/presentations). All data are grouped by projects, and associated with the researchers who produced them. In order to encourage sharing of data we allow researchers flexibility in the formats they upload and share their data in. This means data formats in the FAIRDOM-SEEK catalogue can vary. We do offer a set of “best practice” guidelines for researchers who want to make their data available and usable to the widest possible audience.
```

Comment:
could also add Documents to the list here, and that they can be put into collections. But not a blocker, can be done later.

Also Datasets could link to fairdomhub.org/data_files